src/vs/workbench/browser/parts/activitybar/activitybarActions.ts

```diff
@@ -310,6 +310,10 @@ export class ViewletActionItem extends ActivityActionItem {
 	public render(container: HTMLElement): void {
 		super.render(container);
 
+		if (this._keybinding) {
```

Comment:
@RohithKumbharkar I have a different suggestion where to put this code that makes it much easier. If you look below in that class there is a `public set keybinding(keybinding: string)`. I suggest to set the title in there because it gets computed there:

```typescript
this.$label.title(title);
this.$badge.title(title);
this.$container.title(title); // <-- add it here
```